src/core/js/adapt.js

```diff
@@ -43,33 +46,34 @@ define(function(require){
         
     }
 
-    Adapt.navigateToElement = function(selector, settings) {
+    Adapt.navigateToElement = function(selector, settings, offset) {
         // Allows a selector to be passed in and Adapt will navigate to this element
 
+        if(offset===undefined) offset = -($('.navigation').height()+10);
+
         // Setup settings object
         var settings = (settings || {});
 
         // Removes . symbol from the selector to find the model
         var currentModelId = selector.replace(/\./g, '');
-        var currentModel = Adapt.findById(currentModelId);
+        var currentModel = Adapt[Adapt.mapById(currentModelId)].findWhere({_id: currentModelId});
```

Comment:
findById does Adapt[Adapt.mapById(currentModelId)].findWhere({_id: currentModelId}); so you can revert this line.